enarx-keep-sgx-shim/src/start.S

```diff
@@ -163,6 +197,40 @@ syscall:
 
     xor     %rcx,                   %rcx            # Clear %rcx
     zerof   %rcx                                    # Clear CPU flags
+
+    # Check for unexpected flags in xcr0 before
+    # executing xsavec
+    mov    $0,                      %rcx            # Specify xcr0 for xgetbv
+    xgetbv
+    and    $~0x3FF,                 %eax            # Mask xgetbv lowest 9 bits
```

Comment:
for 0x3FF that is: `lowest 10 bits (0-9 incl.)`

Reply:
Yeah, of course, you're right.